packages/kbn-reporting/common/types.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Ensure, SerializableRecord } from '@kbn/utility-types';
+import type { LayoutParams } from '@kbn/screenshotting-plugin/common';
+import { TypeOf } from '@kbn/config-schema';
```

Comment:
I'd look into these imports first. This might be what is causing client-side bundle issue